### PR TITLE
Forgot to push commit for Tunernome

### PR DESCRIPTION
**Pull request by haloflooder**
Forgot to push out major changes
- Added subdivided beat system

Also put in some bug fixes:
- Changed position of metronome arm and size
- Sync beat systems when changing BPMs
- Fixed spacing between beat circles (cast some vars to a float)

### CODE REVIEW

File: main/modes/mode_tunernome.c [hunk 1]
```diff
@@ -554,6 +563,16 @@ void recalcMetronome(void)
     // Figure out how many microseconds are in one beat
     tunernome->usPerBeat = (60 * 1000000) / tunernome->bpm;
 
+    switch (tunernome->subdividedBeats) {
+        case 0: tunernome->usPerSubBeat = tunernome->usPerBeat; break;
```
Comment (AEFeinstein) [inline main/modes/mode_tunernome.c:567]: Not requesting a change, but you could math this to be a single
```C
tunernome->usPerSubBeat = tunernome->usPerBeat / (1 + tunernome->subdividedBeats);
```

File: main/modes/mode_tunernome.c [hunk 2]
```diff
@@ -353,6 +357,7 @@ static const char playStringsText[] = "Play all open strings";
 static const char listeningText[] = "Listening for a note";
 static const char leftStr[] = ": ";
 static const char rightStrTuner[] = "Start: Tuner";
+static const char rightStrTunerSel[] = "Sel: SubBeat";
```
Comment (AEFeinstein) [inline main/modes/mode_tunernome.c:360]: It would be nice if this listed the number of subbeats similar to how the number of beats are listed.

**Comment by AEFeinstein** [comment]
I was playing around with this on a physical swadge and experienced two crashes. I don't know why, but this branch is not getting merged until the issue is well known and fixed.

**Review by AEFeinstein — CHANGES_REQUESTED** [review]
Identify and fix crash before merge

**Comment by haloflooder** [comment]
Couldn't reproduce the crash on the emulator but it happens on hardware. Decided to leave this feature out during the meeting.

**Pull request closed without merging**